src/javascript/app/pages/user/account/statement/statement.init.js

```diff
@@ -127,6 +126,7 @@ const StatementInit = (() => {
         no_more_data = false;
 
         current_batch = [];
+        filter =  $('#dropdown_statement_filter').val();
```

Comment:
Is behavior here supposed to be the same as on init?

Reply:
This one is the onInitPage and i think it should be implemented inside this function

Reply:
I mean this is `onUnload`, I just wonder why do we need to do the same thing onInit and onUnload